### ISSUE TITLE
ertheia - ertheia tutorial quests

### ISSUE THREAD

**Issue by karma12** [L2J/L2J_DataPack#86]
hello and sry for posting here
1. in quest 10735 when you click at html to enter instance it doesnt let you go in
2. newbie guide buffer casts the buffs on himself(script is fine but buffs are auto-generated)
3. in quest Q10742_AFurryFriend you have to find and bring back to a npc his wolf-type pet. the pet works fine until you get it out from the cave(as ordered by official) but when it has to follow to you, it doesnt; instead it runs back to the cave and the timer starts again and it is f@cked up  :)
4. in quest Q10744_StrongerThanSteel the html 33953-01.html has a mistake. it is 
 <Button ALIGN=LEFT ICON="NORMAL" action="bypass -h Quest Q10744_StrongerThanSteel 33952-01.htm">"Who."</button>
BUT it should be like that -> <Button ALIGN=LEFT ICON="NORMAL" action="bypass -h Quest Q10744_StrongerThanSteel 33953-02.htm">"Who."</button>

by the chance, there's a small mistake on vitality: you can use up to 999 vitality items per week(official) but here the number is 5.
but the most serious issue (for me) is that the spiritshot/soulshot R grade are not working.

i ve seen more mistakes/errors/bugs but i m not able to remember more. sorry for posting here the issues(im newbie), by fixing those issues it ll make the project going better and faster on implementation. thanks :)

**Comment by Zoey76** [comment]
Hello there, thank you for your bug report, in the future please consider to open one issue for each problem, so we can divide and assign them.

**Comment by karma12** [comment]
yeah, no problem, im new to github :)

**Comment by Sdw-** [comment]
Corrected 1 in @6ef369fc1b04830b91e88df71dcd7ae320e1b83c

2. Is a known issues, still aren't done.

3. Happens sometimes, not sure why, requires investigation.

4. I didn't understand anything about that part ?

"Who."
BUT it should be like that -> "Who."

I don't see any differences.

**Comment by karma12** [comment]
haha :) 
something's wrong with the bypass, sorry i havent said it right, 

action="bypass -h Quest Q10744_StrongerThanSteel 33953-02.htm">"Who." 
 is the right one

the commited html was <action="bypass -h Quest Q10744_StrongerThanSteel 33952-01.htm">"Who."

so, using the commited one, when you click the quest button on first html, it doesnt guide you to the second html so as to begin quest

**Comment by karma12** [comment]
also, reffering to faeron island, i noticed something other :P
outside the village there is a lottery npc from the old talking island (c1-Hi5) and it shouldn't exist there:)

**Comment by Gladicek** [comment]
That NPC exist on retail :) But seems like fail by ncsoft cause you cant speak with him. So we can remove it.

**Comment by karma12** [comment]
yes i know that exist as a npc id, but i checked it on shilen server that lottery ticket seller hasnt a spawn point outside faeron village

**Comment by Gladicek** [comment]
Chronos: http://i.imgur.com/uuL0onF.png

**Comment by karma12** [comment]
yes im talking about that npc, havent seen it on shilen so far anyway..

**Comment by UnAfraid** [comment]
I guess someone pushed him to there.
U can push back npcs with some skills like healer's

**Comment by Gladicek** [comment]
I tested that only on Nomi and thats works cause, he is made like that (he spam some npctring id when you push him).
